### PR TITLE
Update CODEOWNERS file

### PR DESCRIPTION
**Pull request by marcos-iov**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1 @@
-  
-*       @rootstock/rsk-fed
+*       @rsksmart/rsk-fed
```
[the remark this answers is not in the document]
Comment (marcos-iov) [reply CODEOWNERS:1]: done

**Review by nathanieliov — APPROVED** [review]

**Review by kelvinator07 — APPROVED** [review]

**Comment by marcos-iov** [comment]
pipeline:run

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Pull request merged**